data/globalevents/scripts/serversave.lua

```diff
@@ -15,18 +15,19 @@ local function serverSave()
 	end
 end
 
+local endingText = shutdownAtServerSave and "Please logout." or "Please mind it may freeze!"
 local function secondServerSaveWarning()
-	broadcastMessage("Server is saving game in one minute. Please logout.", MESSAGE_STATUS_WARNING)
+	broadcastMessage(string.format("Server is saving game in one minutes. %s.", endingText), MESSAGE_STATUS_WARNING)
```

Comment:
This can be reverted altogether since it's inaccurate, every player gets kicked at server save regardless of config.